### DISCUSSION TITLE
Modifiers for modifiers!

### DISCUSSION THREAD

**Discussion by AlenaFis** [olivierfriard/BORIS#511] [Ideas]
To whom it may concern, 

would it be possible to create "submodifiers" for single modifier in one set? I need to specify my modifier sometimes, but the information are good only for one or two of them, not for all, so when I create a new set of modifiers, I need to use it everytime. It would be really nice, if I could use one modifier and BORIS would ask me about the other information just in that one single case, not everytime. 

Best wishes,
Alena

**Comment by olivierfriard** [comment]
Hi Alena,

thank you for your suggestion.
I think that a submodifier function may be a little bit complicated to handle...

Olivier
